pkg/lang/frontend/starlark/v1/interpreter.go

```diff
@@ -42,6 +43,19 @@ type entry struct {
 	err     error
 }
 
+func envdStarlarkResolveOptions() *syntax.FileOptions {
+	return &syntax.FileOptions{
+		// resolver
+		Set:               false,
+		While:             false,
+		TopLevelControl:   false,
+		GlobalReassign:    false,
+		LoadBindsGlobally: false,
+		// compiler
+		Recursion: true,
+	}
```

Comment:
The latest version of starlark requires explicit options instead of legacy global options.

The configurations here should work for most of the cases.